app/policies/person_policy.rb

```diff
@@ -19,7 +19,13 @@ def permitted_attributes(_params)
 
   class Scope < ApplicationScope
     def resolve
-      scope.all
+      return scope.none if person.blank?
+
+      if person.operator?
+        scope.all
+      else
+        scope.where(id: person.id)
```

Comment:
So I can only see myself?

Reply:
Yep! But you can see the `Memberships` for all of your Spaces (defined in the `MembershipPolicy`).

Reply:
(Also, the `show?` policy method above matches this: you can only `show?` yourself if you're not an Operator.)

Reply:
Makes sense! Thank you!